### ISSUE TITLE
Fluent Svelte for SvelteKit does not work at all

### ISSUE THREAD

**Issue by carrved** [Tropix126/fluent-svelte#55]
### Before you start...

- [X] Have you updated your dependencies? You might be using an old version of the library.
- [X] Have you checked for an existing issue on this topic? If there is one, post a comment on it instead.

### What browsers are you seeing the problem on?

Chrome

### Description

Just trying to port a Svelte project to SvelteKit so I could add more pages to it, and when I started copying everything, the css was not being applied at all.

### Steps To Reproduce

1. Make new SvelteKit project using npm (skeleton project)
2. Copy a Svelte project (using Fluent Svelte)'s files over to SvelteKit
3. Nothing works!

### Expected behavior

The css would be applied and the page would look normal.

### Relevant Assets

https://i.imgur.com/MVAG8ks.jpeg
(what the site is supposed to look like)
https://i.imgur.com/68JuLlk.png
(what the site looks like with sveltekit)

**Comment by dongle-the-gadget** [comment]
Yeah, looks like something is going on with the latest SvelteKit that is breaking Fluent Svelte.

**Comment by dongle-the-gadget** [comment]
@z4xi, in case you are still trying to find a solution, add `<slot />` to your `+layout.svelte`. Fluent Svelte isn't broken, the docs are.
